src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -361,6 +361,10 @@ private function createFirewall(ContainerBuilder $container, $id, $firewall, &$a
                 'csrf_parameter' => $firewall['logout']['csrf_parameter'],
                 'csrf_token_id' => $firewall['logout']['csrf_token_id'],
                 'logout_path' => $firewall['logout']['path'],
+                'default_target_path' => isset($firewall['form_login']) ? $firewall['form_login']['default_target_path'] : null,
+                'always_use_default_target_path' => isset($firewall['form_login']) ? $firewall['form_login']['always_use_default_target_path'] : null,
```

Comment:
Shouldn't the default value here as well as for the `use_referer` option be `false` instead of `null`?

Reply:
We're purposely making these options match the `form_login` options? That looks odd to me. Also, I don't think the `LogoutListener` handles these new options... they're passed as the `$options` argument... but those are not used in the class.